content/pt-br/docs/concepts/storage/volume-health-monitoring.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Monitoramento de Saúde do Volume
+content_type: concept
+weight: 100
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+{{< glossary_tooltip text="CSI" term_id="csi" >}} O monitoramento de saúde de volume CSI permite que os drivers CSI detectem condições anormais de volume dos sistemas de armazenamento subjacentes e os relatem como eventos em {{< glossary_tooltip text="PVCs" term_id="persistent-volume-claim" >}} ou {{< glossary_tooltip text="Pods" term_id="pod" >}}.
+
+<!-- body -->
+
+## Monitoramento de Saúde do Volume
+
+_O monitoramento de saúde do volume do Kubernetes_ é uma parte da forma como o Kubernetes implementa a Interface de Armazenamento de Contêiner (CSI). O recurso de monitoramento de saúde de volume é implementado em dois componentes: um controlador de Monitor Externo de Saúde e o {{< glossary_tooltip term_id="kubelet" text="kubelet" >}}.
+
+Se um driver CSI suporta o recurso de monitoramento de saúde do volume do lado do controlador, um evento será relatado no {{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}} (PVC) quando uma condição anormal de volume for detectada em um volume CSI.
+
+O {{< glossary_tooltip text="controlador" term_id="controller" >}} de Monitor Externo de Saúde também observa eventos de falha de nós. Você pode habilitar o monitoramento de falha de nós configurando a flag `enable-node-watcher` como verdadeira. Quando o monitor de saúde externo detecta um evento de falha de nó, o controlador reporta um evento que será relatado no PVC para indicar que os pods que usam este PVC estão em um nó com falha.
```

Comment:
Suggestions here:
* Perhaps using "opção" instead of "flag" would be a good idea.
* Using "`true`" instead of "verdadeira" might be a better idea (since we want to instruct the reader what's the value that they should use).
* Although "report" appears twice in the original sentence, I think you could use another verb instead of "relatado". Perhaps "indicado", or "listado", or another choice of your preference.
* We prefer rendering "Pods" with a capital P to indicate they are a Kubernetes resource.
* Suggestion: add "alocados" after "estão" in the "Pods que usam este PVC..." sentence.

Reply:
I followed the directions and it's fixed.

Reply:
Hi there, @castroricardo1!

I think the "relata"/"relatado" is now "indica"/"indicado", which ends up in the same problem (a redundant verb) 😅
Would you mind changing one of those to a different verb?
Suggestions:
* "reporta um evento que será relatado no PVC para indicar"
* "reporta um evento que será indicado no PVC para relatar"

This is really a nit, but I feel reading the sentence with the same verb twice is a bit strange.

Reply:
yes, it makes sense, I chose the first option, for me it seems to be the best